data_safe_haven/backend/backend.py

```diff
@@ -81,32 +78,25 @@ def create(self) -> None:
                 tags=self.tags,
             )
             if not keyvault.name:
-                raise DataSafeHavenAzureException(
-                    f"Keyvault '{self.config.backend.key_vault_name}' was not created."
-                )
+                msg = f"Keyvault '{self.config.backend.key_vault_name}' was not created."
+                raise DataSafeHavenAzureError(msg)
             pulumi_encryption_key = self.azure_api.ensure_keyvault_key(
                 key_name=self.config.pulumi.encryption_key_name,
                 key_vault_name=keyvault.name,
             )
-            self.config.pulumi.encryption_key_id = pulumi_encryption_key.id.split("/")[
-                -1
-            ]
+            self.config.pulumi.encryption_key_id = pulumi_encryption_key.id.split("/")[-1]
```

Comment:
I prefer this, but `black` imposed the other syntax. What setting have you changed to make sure it won't do so if I run it again?

Reply:
Not certain what would be different between our configurations. The black rules are in `pyproject.toml` so everyone _should_ be able to arrive at the same result by running `hatch run lint:fmt`.